### PR TITLE
 IDVA3-1116 and tidy up

### PR DESCRIPTION
**Pull request by eedwards0**
###  **IDVA3-1116 and general tidy up**
 
- Corrected the URL displayed for the PSC Type Page
-  Maintain the locale when navigating to/from the psc-type page
-  Removed the 'skeleton three' files, as no longer needed
-  Renamed the 'get' function in psc-type handler to match format of the other handlers
-  Updated the psc-type router to match the format of the other routers
- Updated debug statement in individual_statement.ts

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/confirmCompanyRouter.ts [hunk 1]
```diff
@@ -11,9 +13,10 @@ router.get("/", handleExceptions(async (req: Request, res: Response, _next: Next
 }));
 
 router.post("/", handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
-    const handler = new PscTypeHandler();
-    const { templatePath, viewData } = handler.execute(req, res);
-    res.render(templatePath, viewData);
+    // const handler = new PscTypeHandler();
+    // const { templatePath, viewData } = handler.executeGet(req, res);
+    // res.render(templatePath, viewData);
```
Comment (PhilipGlover) [inline src/routers/confirmCompanyRouter.ts:18]: remove the commented out lines, won't be needed

**Comment by eedwards0** [comment]
I've left in the changes for the locales and buttons but agree we need to implement a generic way for this going forward.

File: src/routers/confirmCompanyRouter.ts (hunk 1 above)
Comment (eedwards0) [reply src/routers/confirmCompanyRouter.ts]: Done

**Review by PhilipGlover — APPROVED** [review]
- redirect better
- lang navigation works, but maybe could be cleaner later

**Pull request merged**